proto/errors.proto

```diff
@@ -192,6 +193,11 @@ enum TransactionRestart {
   IMMEDIATE = 2;
 }
 
+// ErrPosition describes the position of an error in a Batch.
+message ErrPosition {
```

Comment:
Is there a value to this message versus just embedding `optional int32 index` in the errors which require it?